desktop/src/main/java/bisq/desktop/components/paymentmethods/data/JapanBankData.java

```diff
@@ -355,7 +355,7 @@ private static String prettyPrintBank(String bankId, String bankName) // {{{
         .put("0562", "みなと銀行")
         .put("0565", "島根銀行")
         .put("0566", "トマト銀行")
-        .put("0569", "もみじ銀行")
+        .put("0569", "もみじ銀行")
```

Comment:
What did you change here? @wiz Is this correct?

Reply:
Hm, that was not intended. Will revert.

Reply:
Some weird encoding issue with editor. Fixed.